membership-attribute-service/app/models/Attributes.scala

```diff
@@ -18,13 +18,15 @@ object ContentAccess {
 }
 
 case class Attributes(
-                       UserId: String,
-                       Tier: Option[String] = None,
-                       MembershipNumber: Option[String],
-                       AdFree: Option[Boolean] = None,
-                       CardExpirationMonth: Option[Int] = None,
-                       CardExpirationYear: Option[Int] = None,
-                       ContributionFrequency: Option[String] = None) {
+  UserId: String,
+  Tier: Option[String] = None,
+  MembershipNumber: Option[String],
+  AdFree: Option[Boolean] = None,
+  CardExpirationMonth: Option[Int] = None,
+  CardExpirationYear: Option[Int] = None,
+  ContributionFrequency: Option[String] = None,
+  MembershipJoinDate: Option[LocalDate] // TODO startDate shouldn't be optional once we've backfilled it everywhere
```

Comment:
thanks, good point, I didn't think about that as part of the merge.  Since some of the fields are optional together rather than individually, we should consider having an Option[Membership] and Option[Contribution] which have all the fields inside as required.